styles/modules/modals/_editListing.scss

```diff
@@ -7,6 +7,36 @@
       min-height: 150px;
     }
 
+    .tile {
+      width: 100px;
+      height: 100px;
+      border: 1px solid;
+      border-color: inherit;
+      border-radius: $corner;
+      float: left;
+      margin-right: $pad;
+      margin-bottom: $pad;
+      position: relative;
+
+      .closeIcon {
+        width: 24px;
+        height: 24px;
+        position: absolute;
+        top: 0;
+        left: 0;
+      }      
+
+      button {
```

Comment:
Even though it's unlikely there will be another button element inside the tile, since the button already has the .btn class, can this just use that class instead?